MonkeyLoader.GamePacks.ResoniteModLoader/MonkeyLoader.GamePacks.ResoniteModLoader.csproj

```diff
@@ -12,7 +12,7 @@
       <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
       <PackageId>MonkeyLoader.GamePacks.ResoniteModLoader</PackageId>
       <Title>ResoniteModLoader</Title>
-      <Authors>Banane9</Authors>
+      <Authors>Banane9, Nytra</Authors>
```

Comment:
Hmmm - the official docs say this should be a semicolon separated list, but it seems it accepts both :'D